app/controllers/synchronizations_controller.rb

```diff
@@ -1,7 +1,21 @@
 class SynchronizationsController < ApplicationController
   def new
-    @facility = Facility.find(params[:facility_id])
-    Fetcher.new(@facility).run
-    redirect_to facilities_path
+    synchronizer.synchronize_data
```

Comment:
`_data` is redundant, of course it will sync data, what else? Everything is data. In this case seems better to just:

```ruby
synchronizer.synchronize
```

We can probably be more concrete:

```ruby
facylity_synchronizer.syncronize_facilities
```